src/db/books.js

```diff
@@ -0,0 +1,59 @@
+// Code dealing with getting our books from a data source
+// 
+
+/**
+ * @typedef {import('./book').Book} Book
+ */
+
+module.exports = {
+  initBooksDb: (db) => ({
+    /**
+     * Returns the books in the db 
+     * @param {number} offset
+     * @param {number} limit
+     * @return {Book[]}
+     */
+    getScifiBooks(offset = 0, limit = 10) {
+      const values = db.getRange({ limit, offset })
+        .map(({ value }) => value);
+      return [...values];
+    },
+
+    /**
+     * Get book from db by some ISBN
+     * @param {string} isbn
+     * @return {Book || null}
+     */
+    getScifiBook(isbn) {
+      return db.get(isbn);
```

Comment:
As you can see, reading from a real database is a WHOLE lot easier to understand